src/wallet/spend.cpp

```diff
@@ -1091,13 +1106,16 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     // Static vsize overhead + outputs vsize. 4 version, 4 nLocktime, 1 input count, 1 witness overhead (dummy, flag, stack size)
     coin_selection_params.tx_noinputs_size = 10 + GetSizeOfCompactSize(vecSend.size()); // bytes for output count
 
+    LogInfo("input count size %s \n", GetSizeOfCompactSize(vecSend.size()));
+
     // vouts to the payees
     for (const auto& recipient : vecSend)
     {
         CTxOut txout(recipient.nAmount, GetScriptForDestination(recipient.dest));
 
         // Include the fee cost for outputs.
         coin_selection_params.tx_noinputs_size += ::GetSerializeSize(txout);
+        LogInfo("Output size %s \n", static_cast<int>(::GetSerializeSize(txout)));
```

Comment:
Same as above, guess that this was used for testing purposes and you forgot to remove it?

Reply:
removed